ee/systray/runtime/runner_test.go

```diff
@@ -0,0 +1,85 @@
+//go:build darwin
+// +build darwin
+
+package runtime
+
+import (
+	"math"
+	"os"
+	"os/exec"
+	"os/user"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSystrayUserProcessRunner_Execute(t *testing.T) {
+	t.Parallel()
+
+	// When running this using the golang test harness, it will leave behind proccess if you do not build the binary first.
+	// On mac os you can find these by setting the executable path to an empty string before running the tests, then search
+	// the processes in a terminal using: ps aux -o ppid | runtime.test after the tests have completed, you'll also see the
+	// CPU consumtion go way up.
+
+	// To get around the issue mentioned above, build the binary first and set it's path as the executable path on the runner.
+	executablePath := filepath.Join(t.TempDir(), "systray")
```

Comment:
```suggestion
	executablePath := filepath.Join(t.TempDir(), "systray.exe")
```

Somewhat optional, but if/when windows, we'll need `.exe`